octodns/source/tinydns.py

```diff
@@ -45,6 +46,37 @@ def _data_for_A(self, _type, records):
             'values': values,
         }
 
+    def _data_for_AAAA(self, _type, records):
+        values = []
+        for record in records:
+            values.append(u":".join(textwrap.wrap(record[0], 4)))
```

Comment:
What's going on with the textwrap stuff here? I'm not grokking what's happening here. It'll definitely need an explanatory comment if it's required, but it's fine to start discussion here first.

Reply:
TinyDNS files have the ipv6 address written in full, but with the colons removed. This just inserts a colon every 4th character to make the address correct.

Will add a comment when I get chance.

Reply:
Ugh. OK. Please add a comment to that effect in the code as it's definitely non-obvious/standard.

Reply:
Will do. Or can you suggest a clearer and perhaps more Pythonic way of doing it?

Reply:
I can't think of anything cleaner and it sounds like `textwrap.wrap` makes sense here. The suprising/weird part to me is that TinyDNS doesn't have `:`. 

> TinyDNS files have the ipv6 address written in full, but with the colons removed. This just inserts a colon every 4th character to make the address correct.

Will work as the comment.

Reply:
Comment added. Let me know if you need anything further.